### PR TITLE
Added removed item name or id to delete dialog message.

### PR DESCRIPTION
**Pull request by michal-damiecki**
UIE-256

screenshot from 2018-04-16 09-29-49

### CODE REVIEW

**Comment by thayashi** [comment]
@michal-damiecki  how about bold the resource name so that easy to see the name?

**Comment by michal-damiecki** [comment]
@thayashi Thanks, good idea. Done.

File: css/form.scss [hunk 1]
```diff
@@ -256,3 +256,7 @@
 .search-container {
   padding: 0.5rem;
 }
+
+.gohan-removed-resource {
+  font-weight: bold;
```
[the remark this answers is not in the document]
Comment (michal-damiecki) [reply css/form.scss]: I know, but I wanted to separate styling from the semantic purpose of `strong` tag. Now, after your comment and some reconsiderations, I agree that it doesn't make much sense. I'll remove it.

**Comment by thayashi** [comment]
LGTM

**Pull request merged**